spanner/integration_test.go

```diff
@@ -3458,7 +3501,7 @@ func prepareDBAndClient(ctx context.Context, t *testing.T, spc SessionPoolConfig
 	if _, err := op.Wait(ctx); err != nil {
 		t.Fatalf("cannot create testing DB %v: %v", dbPath, err)
 	}
-	if dbDialect == adminpb.DatabaseDialect_POSTGRESQL {
+	if dbDialect == adminpb.DatabaseDialect_POSTGRESQL && len(statements) > 0 {
```

Comment:
When will `statements` is empty?

Reply:
I see. For `TestIntegration_InitSessionPool`, the statements is empty. I am not sure if we want to run tests, like `TestIntegration_InitSessionPool`, twice for different dialects.

Reply:
I am ok to remove test TestIntegration_InitSessionPool, but validating params(empty statements) before calling updateDDL will save us making extra call, also I think by testing with PG dialect TestIntegration_InitSessionPool we are making sure that behaviour of sessions is same with both dialect and any future issues will be caught by this test.